plugin/gudid-transformation-maven-plugin/src/main/java/dev/ikm/maven/DeviceTransformer.java

```diff
@@ -52,69 +48,46 @@ public void transform(File inputFile, Composer composer) {
         EntityProxy.Concept path = TinkarTerm.DEVELOPMENT_PATH;
         EntityProxy.Concept module = gudidUtility.getModuleConcept();
 
+        EntityProxy.Concept publicDeviceRecordKey = EntityProxy.Concept.make("Public Device Record Key", UuidT5Generator.get(namespace, "Public Device Record Key"));
```

Comment:
When we create a concept proxy like we do here for publicDeviceRecordKey, at some point we have to create the concept (via the Composer api), or the proxy has to refer to an already created concept. My guess is this is the cause of the null entity in komet that you are seeing.